src/ui/public/agg_types/metrics/percentile_ranks.js

```diff
@@ -49,11 +49,9 @@ export default function AggTypeMetricPercentileRanksProvider(Private) {
       return fieldFormats.getInstance('percent') || fieldFormats.getDefaultInstance('number');
     },
     getValue: function (agg, bucket) {
-      // values for 1, 5, and 10 will come back as 1.0, 5.0, and 10.0 so we
-      // parse the keys and respond with the value that matches
-      return _.find(bucket[agg.parentId] && bucket[agg.parentId].values, function (value, key) {
-        return agg.key === parseFloat(key);
-      }) / 100;
+      const values = bucket[agg.parentId] && bucket[agg.parentId].values;
+      const percentile = _.find(values, value => agg.key === value.key);
+      return percentile ? percentile.value / 100 : null;
```

Comment:
The existing behavior would always return a number, can you think of any circumstances where returning `null` would cause issues?